zkevm-circuits/src/evm_circuit/execution/error_invalid_jump.rs

```diff
@@ -38,8 +40,18 @@ impl<F: Field> ExecutionGadget<F> for ErrorInvalidJumpGadget<F> {
     const EXECUTION_STATE: ExecutionState = ExecutionState::ErrorInvalidJump;
 
     fn configure(cb: &mut EVMConstraintBuilder<F>) -> Self {
-        let code_len = cb.query_cell();
-        let dest = WordByteCapGadget::construct(cb, code_len.expr());
+        let is_first_bytecode_table = cb.query_cell();
+        #[cfg(not(feature = "dual_bytecode"))]
+        let code_len_gadget = BytecodeLengthGadget::construct(cb, cb.curr.state.code_hash.clone());
+
+        #[cfg(feature = "dual_bytecode")]
+        let code_len_gadget = BytecodeLengthGadget::construct(
+            cb,
+            cb.curr.state.code_hash.clone(),
+            is_first_bytecode_table.expr(),
+        );
+
+        let dest = WordByteCapGadget::construct(cb, code_len_gadget.code_length.expr());
```

Comment:
i woudl suggest move `is_first_bytecode_table` cell into BytecodeLengthGadget? So we can make config and assign looks more clean?

Reply:
for most opcode gadgets,  `SameContextGadget` already contains `is_first_bytecode_table` cell.  `BytecodeLengthGadget` construction can just pass `SameContextGadget.is_first_bytecode_table` as parameter instead of hold another one. only few case, like 'STOP' and some err cases don't use `SameContextGadget`, so add `is_first_bytecode_table` cell explicitly.